src/azure-openapi-validator/tests/basic-test.ts

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+
+@suite class Hello {
```

Comment:
We don't, this really is sort of a tutorial-ish file until everyone gets comfortable with writing these tests.